bundles/org.openhab.binding.sinope/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file seems copied to old ESH-INF directory by mistake? File already exists in OH-INF.